commands/scanpullrequest.go

```diff
@@ -243,8 +223,8 @@ func getFullPathWorkingDirs(workingDirs []string, baseWd string) []string {
 
 func auditTarget(scanSetup *utils.ScanDetails) (res []services.ScanResponse, isMultipleRoot bool, err error) {
 	// First download the target repo to temp dir
-	log.Info("Auditing ", scanSetup.Git.RepoName, scanSetup.Branch)
-	wd, cleanup, err := utils.DownloadRepoToTempDir(scanSetup.Client, scanSetup.Branch, scanSetup.Git)
+	log.Info("Auditing ", scanSetup.Git.RepoName, scanSetup.Branch())
```

Comment:
Redunfant white-space after "Auditing" (the log.Info function adds white-spaces on its own).
I actually suggest -

log.Info("Auditing ", scanSetup.Git.RepoName, scanSetup.Branch())
-->
log.Info("Auditing repository:", scanSetup.Git.RepoName, "branch:", scanSetup.Branch(), "...")

Reply:
Done